### PR TITLE
1995 intro edit

### PR DESCRIPTION
**Pull request by cvalarida**
Reference department-of-veterans-affairs/vets.gov-team#1096

image

### CODE REVIEW

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -4,37 +4,79 @@ import { focusElement } from '../../../common/utils/helpers';
...
+            </li>
+            <li className="step four last">
+              <div><h5>Decision</h5></div>
+              <ul><li>We usually process claims within 14 days.</li></ul>
+              <ul><li>You'll get a COE or Award Letter in the mail if your application was approved. Bring this to the VA certifying official at your school.</li></ul>
+            </li>
+          </ol>
         </div>
-        <div className="row form-progress-buttons">
+        <div className="row progress-box progress-box-schemaform form-progress-buttons schemaform-buttons">
```
Comment (cvalarida) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: This seems a little ridiculous, but it was the easiest way to get the buttons to be `width: 100%` that I could see without adding a new class for it. Open to suggestions.

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -4,37 +4,79 @@ import { focusElement } from '../../../common/utils/helpers';
...
-  goForward() {
+  goForward = () => {
     this.props.router.push(this.props.route.pageList[1].path);
   }
+  goBack = () => {
+    // Wasn't able to figure out how to get react-router to go to
+    //  /education/apply-for-education-benefits/ without having it prepend the
+    //  pathname with /education/apply-for-education-benefits/application/
+    // this.props.router.replace('/education/apply-for-education-benefits/');
+    window.location = "/education/apply-for-education-benefits/";
```
Comment (cvalarida) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: If anybody can figure out how to get the router to stop prepending the path with `/education/apply-for-education-benefits/application/`, I'd rather use the router.

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx [hunk 3]
```diff
@@ -4,37 +4,79 @@ import { focusElement } from '../../../common/utils/helpers';
...
 class IntroductionPage extends React.Component {
-  constructor() {
-    super();
-    this.goForward = this.goForward.bind(this);
-  }
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
-  goForward() {
+  goForward = () => {
```
Comment (cvalarida) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: We can do this rather than `this.goForward = this.goForward.bind(this);` in the `super()` to save space. Huzzah for ES6!

If the other way was used for a reason, just let me know.

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx (hunk 1 above)
Comment (jbalboni) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: There's some styling that assumes a progress-box class. I'm not sure why it exists, but yeah, this is probably easier for now. We should see if there's an easy way to re-write the rules that use progress-box.

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx (hunk 2 above)
Comment (jbalboni) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: The router won't work for this, because the page we're going to is outside of our edu-benefits app.

This should work, but If possible, we should use a regular link for this back button. Something like:

```
<a href="/education/apply-for-education-benefits/application/">
  <button className="usa-button-outline">« Back</button>
</a>
```

File: src/sass/edu-benefits.scss [hunk 4]
```diff
@@ -311,9 +311,17 @@ form.rjsf {
 .schemaform-back-buttons {
   margin-bottom: 7rem !important;
 }
+
 .schemaform-intro {
   padding-bottom: 1rem;
   margin-bottom: 7rem;
+
+  // To be moved into modules/_m-schemaform.scss when it's merged
+  .process {
```
Comment (jbalboni) [inline src/sass/edu-benefits.scss]: Why is this necessary?

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx [hunk 5]
```diff
@@ -4,37 +4,79 @@ import { focusElement } from '../../../common/utils/helpers';
...
+                <li>Social Security number (required)</li>
+                <li>Military history (required)</li>
+                <li>Basic information about the school or training facility where you want to attend (required)</li>
+                <li>Bank account direct deposit information</li>
+                <li>Education history</li>
+              </ul>
               <div className="usa-alert usa-alert-info">
                 <div className="usa-alert-body">
-                  <span><strong>You won’t be able to save your work or come back to finish</strong>. So before you start, it’s a good idea to gather information about your military and education history, and the school you want to attend.</span>
+                  <span><strong>You won't be able to save your work or come back to finish.</strong> So before you start, it's a good idea to gather information about your military and education history, and the school you want to attend.</span>
```
Comment (jbalboni) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: Could switch these quote marks back to the characters used in the old page.

File: src/sass/edu-benefits.scss (hunk 4 above)
Comment (cvalarida) [inline src/sass/edu-benefits.scss]: Without shifting the `ol` over to the left again, it created an uncomfortably large space not present in the screenshot:

---
image

---
Alternatively, I could have moved the title, text under the title, and buttons over to the right to close the gap, and that would have matched the screenshot referenced, but I didn't think that was intended since it would unalign the content with the banner, breadcrumbs, et al.

File: src/js/edu-benefits/1995/components/IntroductionPage.jsx (hunk 2 above)
Comment (cvalarida) [inline src/js/edu-benefits/1995/components/IntroductionPage.jsx]: Sure thing. I didn't want to do that initially because it'll have a slightly different hover behavior (the link location will appear at the bottom of the page), but I can certainly make that happen.

File: src/sass/edu-benefits.scss (hunk 4 above)
Comment (jbalboni) [inline src/sass/edu-benefits.scss]: Gotcha. Can we add a separate class for this, instead of nesting the styles underneath `.schemaform-intro`? `schemaform-intro-process` or something.

I want us to avoid nesting as much as possible. It leads to things like the issue with progress-box above.

**Review by jbalboni — APPROVED** [review]
LGTM. Should get Alex or Molly to ok it.

**Comment by mollyblake** [comment]
A few changes:  
[1] This application is equivalent to Form 22-1995 (Request for Change of Program or Place of Training) and Form 22-5495 (Dependents' Request for Change of Program or Place of Training). 

[2] See what benefits you'll get at the school you want to attend. `[Use` the GI Bill Benefits Tool]/gi-bill-comparison-tool/).

**Comment by cvalarida** [comment]
With the changes @mollyblake mentioned:

image
